strapi/strapi
# [marketplace] add providers tabs

What does it do?
Adds tabs for the marketplace to support Plugins and Providers

Why is it needed?
To make it easy to distinguish between plugins and providers

How to test it?
Go the Markateplace page and select the different tabs:

You should only see plugins on the plugins tab and providers on the providers tab

Note:
I realized we didn't consider how to handle installed providers so aren't marked as such in this PR.

@remidej I think we should discuss and handle it in another PR based off this one so we can keep moving forward with the search as well.
